Patches/GTest/Patch.cmake

```diff
@@ -0,0 +1,12 @@
+#+
+# This file is called as CMake -P script for the patch step of
+# External_GTest.cmake.
+# GTest_patch and GTest_source are defined on the command line along with the
+# call.
+#-
+
+message("Patching GTest")
```

Comment:
Probably best to make this message STATUS, but I don't know how important that is in this context.